metrics/ci_integrations/test/client/buildkite/test_utils/mock/buildkite_mock_server.dart

```diff
@@ -128,10 +129,15 @@ class BuildkiteMockServer extends ApiMockServer {
     return const BuildkiteBuildStateMapper().map(state);
   }
 
-  /// Generates a list of [BuildkiteBuild]s with the given [state].
+  /// Generates a list of [BuildkiteBuild]s with the given [state]
+  /// and [finishedFrom].
```

Comment:
Should we update the documentation of the `_pipelineBuildsResponse` method to include info about the `finishedFrom` parameter?